.github/workflows/ci.yml

```diff
@@ -4,7 +4,9 @@ on:
   merge_group:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
-    branches: [main]
+    branches:
+      - "main"
+      - "release-candidate"
```

Comment:
Would it make sense to also run `gpu-ci` and `merge-tests` for release candidates?

Reply:
Yes, though this is intended as a working branch converging to a working status. E.g. the release of circom-scotia used here is ~1h old.

Things running are valuable. Blocking merges on these not so.